pkg/core/native/crypto_test.go

```diff
@@ -29,6 +29,80 @@ func TestSha256(t *testing.T) {
 		require.Equal(t, "47dc540c94ceb704a23875c11273e16bb0b8a87aed84de911f2133568115f254", hex.EncodeToString(c.sha256(ic, []stackitem.Item{stackitem.NewByteArray([]byte{1, 0})}).Value().([]byte)))
 	})
 }
+func TestKeccak256(t *testing.T) {
+	c := newCrypto()
+	ic := &interop.Context{VM: vm.New()}
+
+	t.Run("bad arg type", func(t *testing.T) {
+		require.Panics(t, func() {
+			c.keccak256(ic, []stackitem.Item{stackitem.NewInterop(nil)})
+		})
+	})
+	t.Run("good", func(t *testing.T) {
+		// 0x0100 hashes to 628bf3596747d233f1e6533345700066bf458fa48daedaf04a7be6c392902476
```

Comment:
Create the list of test-cases, each test-case is a structure, then in a single loop iterate through testcases. This allows to same some repetitive code. Take a look at TestStdLibItoaAtoi.